microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -111,45 +115,69 @@ def _install(self) -> bool:
         self.dpdk_path = self.node.working_path.joinpath(self._dpdk_repo_path_name)
         if result.exit_code == 0:  # tools are already installed
             return True
-        if self._install_dependencies():
-            node = self.node
-            git_tool = node.tools[Git]
-            echo_tool = node.tools[Echo]
+        self._install_dependencies()
+        node = self.node
+        git_tool = node.tools[Git]
+        echo_tool = node.tools[Echo]
+        wget_tool = node.tools[Wget]
+        tar_tool = node.tools[Tar]
```

Comment:
move it in to the if clause, so the tool doesn't need to be created, if it's not needed.